### ISSUE TITLE
Possible to release the fixes to JSON parsing?

### ISSUE THREAD

**Issue by bjarkehs** [prettier/prettier#2379]
Hi,

Is there any chance you guys can release the fixes to JSON parsing made in #2335 ?

It's a little annoying to edit JSON files currently.

**Comment by josephfrazier** [comment]
While you're waiting on this, you can use prettier from github with

```bash
yarn add --dev https://github.com/prettier/prettier
# or
npm install --save-dev https://github.com/prettier/prettier
```

**Comment by benfrain** [comment]
@josephfrazier I tried running `sudo npm install -g https://github.com/prettier/prettier` but it errors:

```
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/eslint/typescript-eslint-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-eslint-typescript-eslint-parser-git-cfddbfe3ebf550530aef2f1c6c4ea1d9e738d9c1-9011120a: Cloning into bare repository '/Users/benfrain/.npm/_git-remotes/git-github-com-eslint-typescript-eslint-parser-git-cfddbfe3ebf550530aef2f1c6c4ea1d9e738d9c1-9011120a'...
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/eslint/typescript-eslint-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-eslint-typescript-eslint-parser-git-cfddbfe3ebf550530aef2f1c6c4ea1d9e738d9c1-9011120a: fatal: unable to connect to github.com:
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/eslint/typescript-eslint-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-eslint-typescript-eslint-parser-git-cfddbfe3ebf550530aef2f1c6c4ea1d9e738d9c1-9011120a: github.com[0: 192.30.253.112]: errno=Operation timed out
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/eslint/typescript-eslint-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-eslint-typescript-eslint-parser-git-cfddbfe3ebf550530aef2f1c6c4ea1d9e738d9c1-9011120a: github.com[1: 192.30.253.113]: errno=Operation timed out
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/shellscape/postcss-values-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-shellscape-postcss-values-parser-git-5e351360479116f3fe309602cdd15b0a233bc29f-321f5184: Cloning into bare repository '/Users/benfrain/.npm/_git-remotes/git-github-com-shellscape-postcss-values-parser-git-5e351360479116f3fe309602cdd15b0a233bc29f-321f5184'...
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/shellscape/postcss-values-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-shellscape-postcss-values-parser-git-5e351360479116f3fe309602cdd15b0a233bc29f-321f5184: fatal: unable to connect to github.com:
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/shellscape/postcss-values-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-shellscape-postcss-values-parser-git-5e351360479116f3fe309602cdd15b0a233bc29f-321f5184: github.com[0: 192.30.253.112]: errno=Operation timed out
npm ERR! git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/shellscape/postcss-values-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-shellscape-postcss-values-parser-git-5e351360479116f3fe309602cdd15b0a233bc29f-321f5184: github.com[1: 192.30.253.113]: errno=Operation timed out
npm ERR! Darwin 16.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "https://github.com/prettier/prettier"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! code 128

npm ERR! Command failed: git clone --template=/Users/benfrain/.npm/_git-remotes/_templates --mirror git://github.com/eslint/typescript-eslint-parser.git /Users/benfrain/.npm/_git-remotes/git-github-com-eslint-typescript-eslint-parser-git-cfddbfe3ebf550530aef2f1c6c4ea1d9e738d9c1-9011120a
npm ERR! Cloning into bare repository '/Users/benfrain/.npm/_git-remotes/git-github-com-eslint-typescript-eslint-parser-git-cfddbfe3ebf550530aef2f1c6c4ea1d9e738d9c1-9011120a'...
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.253.112]: errno=Operation timed out
npm ERR! github.com[1: 192.30.253.113]: errno=Operation timed out
npm ERR!
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/benfrain/.npm/_logs/2017-07-06T11_15_13_690Z-debug.log
```

**Comment by azz** [comment]
@benfrain 

```
fatal: unable to connect to github.com
```

Looks like an internet connection issue. Do you need a proxy set? 
https://docs.npmjs.com/misc/config#proxy

**Comment by marvinhagemeister** [comment]
A patch release would be awesome. `vscode-prettier` even went so far as to fall back to prettier  `1.4.x` when dealing with json files.

**Comment by levithomason** [comment]
Found this issue as I was about to post an issue about regular releases.  Would be superb to have the latest patches/features shipped regularly.  Spent quite a while trying to figure out why multiple globs weren't working (see #2356).

>_Sidenote: @benfrain `npm i -g prettier/prettier` works here._

**Comment by vjeux** [comment]
@azz or @josephfrazier, would you be interested in sending a patch release with just this fix?

**Comment by josephfrazier** [comment]
Can we just release v1.5.3 (or maybe v1.6.0) with [all the changes on master](https://github.com/prettier/prettier/compare/1.5.2...master), or are there outstanding issues?

**Comment by vjeux** [comment]
I'm not at home until July 22 and there's been a lot of major changes since we last released, so this needs to be heavily tested.

**Comment by azz** [comment]
Just created a hotfix branch, I'm happy to push it to npm.

https://github.com/prettier/prettier/compare/1.5.2...hotfix-1.5.3

**Comment by vjeux** [comment]
@azz just added you on the npm repo.

**Comment by azz** [comment]
Published 1.5.3.

**Comment by marvinhagemeister** [comment]
Awesome of you guys for taking the time to make a patch release so quickly! Thank you so much 💯  @vjeux @azz
